src/diffusers/utils/testing_utils.py

```diff
@@ -45,6 +45,21 @@ def torch_all_close(a, b, *args, **kwargs):
     return True
 
 
+def print_tensor_test(tensor, filename="test_corrections.txt", expected_tensor_name="expected_slice"):
```

Comment:
BTW @pcuenca @patil-suraj @yiyixuxu @williamberman 

I added some functionality that helps to automatically correct slice-comparison tests. 
All you need to do here is to add:

```py
print_tensor_test(image_slice)
```

and it'll write the correct tensor to a file: `filename` with the name `expected_tensor_name`.

The utils file here: https://github.com/huggingface/diffusers/pull/1924/files#diff-bd9ba404f47129f2218fc94c5e528cc81bfd10cef1eff1d6d543f6ffa0143367 can then take this file and will automatically overwrite the test with the correct results. This should save quite some time going forward :-)

Reply:
That's very cool, thanks a lot !